### DISCUSSION TITLE
Can't close remoting_host.exe

### DISCUSSION THREAD

**Discussion by LucasNiesen** [SafeExamBrowser/seb-win-refactoring#613] [Q&A]
When i start seb i get a notification to close 1 app but seb can't close it.

Somethimes when i close the remoting_host.exe by me self it will work.
But adter a coupple of seccons it will restart and a get a red screen and the teacher need to enter the password.

This is chrome remote desktop but is there a way so i can solve this.
![IMG_20230514_130748](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/46261961/bc17a11e-9fd4-4ace-83ec-f0c7b36234bf)
![IMG_20230514_130756](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/46261961/839df4c1-1f9e-4526-95e6-43b1c1aa9017)
![IMG_20230514_130803](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/46261961/b0ca77df-25d4-4378-8a0a-dd7b224f5fad)

Here are the logs:
[2023-05-14_13h07m17s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11471775/2023-05-14_13h07m17s_Runtime.log)
[2023-05-14_13h07m17s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11471776/2023-05-14_13h07m17s_Service.log)
[2023-05-14_13h07m17s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11471777/2023-05-14_13h07m17s_Client.log)

**Version Information**
 - OS: WIN 11 22H2 22621.1555
 - SEB-Version 3.5.0.544

**Comment by dbuechel** [comment]
You'll need to make sure that Chrome Remote Desktop isn't running nor restarted automatically. If you do not need Chrome Remote Desktop, consider uninstalling it.

**Comment by LucasNiesen** [comment]
When i close it it auto restart

**Reply by dbuechel** [reply]
Then you need to find out why it automatically restarts or consider uninstalling it, as already stated above.
